task/runner.go

```diff
@@ -490,11 +494,22 @@ func humanizeCatalystError(err error) error {
 		}
 	}
 
+	invalidVideoErrs := []string{
+		"doesn't have video that the transcoder can consume",
+		"is not a supported input video codec",
+		"is not a supported input audio codec",
+	}
+
 	// MediaConvert pipeline errors
-	if strings.Contains(errMsg, "doesn't have video that the transcoder can consume") {
-		// TODO(yondonfu): Add link in this error message to a page with the input codec/container support matrix
-		return errors.New("invalid video file codec or container, check your input file against the input codec and container support matrix")
-	} else if strings.Contains(errMsg, "Failed probe/open") {
+	for _, e := range invalidVideoErrs {
+		if strings.Contains(errMsg, e) {
```

Comment:
Seems like it's getting worth to have a little helper here like `containsAny(...)`